### ISSUE TITLE
dotnet-compile --native not working on OS X

### ISSUE THREAD

**Issue by blackdwarf** [dotnet/cli#841]
Repro steps:

Install PKG
```shell
dotnet new
dotnet restore
dotnet build --native --cpp
```

This gives the following error:

```
Compiling new for DNXCore,Version=v5.0

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:01.3454388
Input Assembly: /Users/zlatkoknezevic/Projects/samples/new/bin/Debug/dnxcore50/new.dll
AppDepSDK Directory does not exist: /usr/local/bin/appdepsdk.
```

**Comment by brthor** [comment]
Root cause here is likely https://github.com/dotnet/coreclr/issues/2128

**Comment by blackdwarf** [comment]
@brthor we can. However, they are not pretty. :)

**Comment by piotrpMSFT** [comment]
Fixed by #840
